src/routes/admin/sources.ts

```diff
@@ -65,25 +69,24 @@ const SourceSchema = {
     }
 };
 
-export function buildAdminSourcesRoutes(deps) {
+export function buildAdminSourcesRoutes(deps: DependenciesLoader) {
     const {
         sourcesService,
         mappingsService,
         targetsService,
         responsesService
     } = deps(['sourcesService', 'mappingsService', 'targetsService', 'responsesService']);
 
-    async function listSources(request, reply) {
+    async function listSources(request: FastifyRequest, reply: FastifyReply<ServerResponse>) {
```

Comment:
I must review scheduler code. I think I am not using this Fastify types. Good point.